dhis2/app-platform
# feat(graphql): create babel plugin to convert gql to app-runtime queries /wo mutations

Fixes LIBS-158

:heavy_check_mark: Done
* Can convert queries (not mutations) from gql to app-runtime queries
* Adds support for the `*` field with the `__all` property (see tests)
* Tested with jest
* Make sure the plugin works when used with babel
* Add the plugin to the app platform
* Iteration 1: Support for queries, but not for mutations

:question: Open questions
* Do we need to add other fields, similar to the `__all` field? (I remember @amcgee mention `:owner`, but I've never heard of / used that field so far
